financial-connections-lite/src/main/java/com/stripe/android/financialconnections/lite/FinancialConnectionsLiteViewModel.kt

```diff
@@ -0,0 +1,39 @@
+package com.stripe.android.financialconnections.lite
+
+import androidx.lifecycle.SavedStateHandle
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.stripe.android.financialconnections.launcher.FinancialConnectionsSheetActivityArgs
+import com.stripe.android.financialconnections.launcher.FinancialConnectionsSheetActivityArgs.Companion.EXTRA_ARGS
+import com.stripe.android.financialconnections.lite.FinancialConnectionsLiteViewModel.ViewEffect.OpenAuthFlowWithUrl
+import com.stripe.android.financialconnections.lite.repository.FinancialConnectionsLiteRepository
+import kotlinx.coroutines.flow.MutableSharedFlow
+import kotlinx.coroutines.launch
+
+internal class FinancialConnectionsLiteViewModel(
+    val savedStateHandle: SavedStateHandle,
+    val repository: FinancialConnectionsLiteRepository,
+    applicationId: String
+) : ViewModel() {
+
+    private val args: FinancialConnectionsSheetActivityArgs
+        get() = savedStateHandle.get<FinancialConnectionsSheetActivityArgs>(EXTRA_ARGS)!!
+
+    internal val viewEffects = MutableSharedFlow<ViewEffect>()
+
+    init {
+        viewModelScope.launch {
+            val sync = repository.synchronize(
```

Comment:
Does this handle exceptions?

Reply:
coming in a followup PR! this is just showcasing how launching the flow will look like.

Reply:
Nice! That probably also applies to some of the other comments?